apps/tailwind-components/components/input/GUIDELINES.md

```diff
@@ -24,3 +24,7 @@ N.B. the following files are not inputs and should be moved elsewhere
 
 - Label
 - Placeholder
+
+Questions:
+
+- do we need required on input atoms? I would expect the form to handle this
```

Comment:
Required and other attributes (e.g, `aria-describedby`, `aria-invalid`, etc.) are important for ensuring an input component can be correctly interpreted by all users. I think we could simply the components by moving these attributes to the new `Field` component, but we need to make sure all elements (e.g., labels, descriptions, errors, etc.) are properly linked to the input element.

Reply:
Added to technical document